contrib/pg_tde/README.md

```diff
@@ -162,4 +162,6 @@ The extension provides the following helper functions:
 
 ### pg_tde_is_encrypted(tablename)
 
-Returns `t` if the relation is encrypted, or `f` otherwise.
+Returns `t` if the relation is encrypted, if plain text `f` or `NULL` if the
```

Comment:
What does it mean "if plain text `f`"? Do you want to say that `f` is returned if the table is not encrypted?
If I understood correctly, the options are:
`t` - the relation is encrypted
`f` - the relation is not encrypted
`NULL` - the relation lacks storage like views, foreign tables, and partitioned
tables and indexes.

Reply:
Correct!

Reply:
Made it cleaerer